packages/edge/android/src/main/java/com/adobe/marketing/mobile/reactnative/edge/RCTAEPEdgeDataBridge.java

```diff
@@ -0,0 +1,99 @@
+/*
+Copyright 2021 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+package com.adobe.marketing.mobile.reactnative.edge;
+
+import android.util.Log;
+
+import com.adobe.marketing.mobile.EdgeEventHandle;
+import com.adobe.marketing.mobile.ExperienceEvent;
+import com.facebook.react.bridge.ReadableArray;
+import com.facebook.react.bridge.ReadableMap;
+import com.facebook.react.bridge.ReadableType;
+import com.facebook.react.bridge.WritableMap;
+import com.facebook.react.bridge.WritableNativeMap;
+
+import java.util.Map;
+
+public final class RCTAEPEdgeDataBridge {
```

Comment:
Same question here. Can this be package protected?

Reply:
We carry over the format from ACP react native and all the extensions are setting it as public right now.  We can raise a issue ticket to look into change this to protected later.

Reply:
Same comment here, please revisit all utils before final release and reduce the visibility.